exercises/space-age/canonical-data.json

```diff
@@ -80,6 +80,16 @@
         "seconds": 1821023456
       },
       "expected": 0.35
+    },
+    {
+      "uuid": "57b96e2a-1178-40b7-b34d-f3c9c34e4bf4",
+      "description": "age on Pluto",
+      "property": "age",
+      "input": {
+        "planet": "Pluto",
```

Comment:
Building on what was said above https://github.com/exercism/problem-specifications/pull/1892/files#r761412772 I would additionally recommend to use some other name than Pluto here to avoid confusion for people that didn't follow the whole "Is Pluto a planet or not?" discussion some time ago. Maybe something like "Milky Way" ...